package.json

```diff
@@ -40,7 +40,7 @@
     "firebase-token-generator": "^2.0.0",
     "istanbul": "0.4.0",
     "mocha": "2.3.3",
-    "mockery": "1.4.0"
+    "proxyquire": "^1.7.3"
```

Comment:
I don't know why, but mockery did not like `mocha --watch`, switching to proxyquire solved the problem.